plugins/github-actions/src/components/Cards/RecentWorkflowRunsCard.tsx

```diff
@@ -70,15 +72,19 @@ export const RecentWorkflowRunsCard = ({
       }
     />
   ) : (
-    <TableContainer component={Card}>
+    <InfoCard
+      title="Recent Workflow Runs"
+      subheader={branch ? `Branch: ${branch}` : 'All Branches'}
+      noPadding
+      variant={variant === 'grid-item' ? 'height100' : ''}
```

Comment:
Was thinking we could add the `'grid-item'` variant to `InfoCard` instead, and remove `'height100'` altogether

Reply:
Idd, since `'height100'` it's not used except in this PR, we could do the change here. But what about the InfoCards used in external plugins? Should we be at least backwards compatible and duplicate the variant for few releases?

Reply:
Yeah let's make `'height100'` a synonym for  '`grid-item`' but print a deprecation warning if it's used.

@spotify/backstage-core, how do you feel about this change and the `'grid-item'` rename? Just wanna double check again before we push this through, as it would be a bunch of hassle to rename again 😅

Reply:
Yeah I'm fine with that naming I guess. I mean, height100 does convey what the visual outcome is, buuuut...

Reply:
.... buuuut?

Idea is that `'grid-item'` conveys something that we can more easily tweak in the future. As in a card that is supposed to exist as an item within a grid. If we later want to change how we organize paddings or whatnot, it's better to apply it to those.

It's also more easy to find as a developer using the API I feel. It's not obvious that you should always use the variant `'height100'` for `InfoCard`s that you place in a grid, but `'grid-item'` makes it more clear.

Reply:
Yeah naming is hard, i'm not opposing really. I suggested just plain "flex" below but maybe it's too limited

Reply:
Yeah tricky thing with doing something like `flex` (or plain `gridItem` for that matter), is that you no longer control whether additional variants are mutually exclusive, at least not through the API. Tbh I'm not sure 100% about the API either, but I think something better would require considerable effort to go through and catalog usages of the `InfoCard` and map out how and where it's being used. My feeling is that this is in the right direction though, and that we would end up adding a few semantic variants and be done there.

Reply:
It's a pity that there are variants at all, is my hot take. But maybe there's absolutely no way around it.